README.md

```diff
@@ -3,45 +3,7 @@
 ## Workshop description
 In this hands-on workshop, we'll refactor a typical data science notebook into a well-structured Kedro project, exploring its core features and workflow and software engineering best practices.
 
-## Requirements
-* PyLadies Amsterdam uses [uv](https://docs.astral.sh/uv/) for dependency management
-* Google account if you want to use [Google Colab](https://colab.research.google.com/)
- 
-## Usage
-### with uv
-Run the following code:
-```bash
-git clone <github-url-of-workshop-repo>
-cd <name-of-repo>
-
-# create and activate venv, install dependencies
-uv sync
-```
-### with Google Colab
-1. Visit [Google Colab](https://colab.research.google.com/)
-2. In the top left corner select "File" &#8594; "Open Notebook"
-3. Under "GitHub", enter the URL of the repo of this workshop
-4. Select one of the notebooks within the repo.
-5. At the top of the notebook, add a Code cell and run the following code:
-```bash
-!git clone <github-url-of-workshop-repo>
-%cd <name-of-repo>
-!pip install -r requirements.txt
-```
-### for a workshop giver
-To get started, open the `pyproject.toml` file and set the required Python version. The pre-selected version 3.8 is generally a safe choice for most use cases.
-
-After you have specified the Python version, you can create a virtual environment with `uv venv` and add packages with `uv add <package>`. Before the workshop, you can generate a requirements.txt file, which is needed e.g. for running code in Google Colab, by running `uv export > requirements.txt`.
-
-## Video record
```

Comment:
Please do not remove Video record and Credits (lines 36-40). The rest can be safely removed.

Reply:
@merelcht last edit required for the README.md

Reply:
Added it back!